src/workflow.ts

```diff
@@ -80,6 +80,143 @@ export async function startWorkflow() {
     process.exit(0);
   }
 
+  // copt the initialResults
+  let result = { ...initialResult };
+
+  try {
+    let confirmOptions = true;
+
+    // loop until they confirm they want to make the project
+    while (confirmOptions) {
+
+      // confirm prompt
+      const { confirm } = await prompts(
+        [
+          {
+            type: "confirm",
+            name: "confirm",
+            message: "Do you want to make any changes?(Default is No)",
+            initial: false,
+          },
+        ],
+        {
+          onCancel: () => {
+            throw new Error(red("✖") + " Operation cancelled");
+          },
+        }
+      );
+
+      if (confirm) {
+        // choose the option prompt
+        const changeOptions = await prompts(
```

Comment:
for a better readability lets move this logic into its own file, `rechoseWorkflow.ts`